subiquity/ui/views/ubuntu_pro.py

```diff
@@ -832,7 +832,7 @@ def itemize(item: str, marker: str = "•") -> Columns:
             Text(""),
             Pile([itemize(svc, marker="  •") for svc in services]),
             Text(""),
-            Text(_("Ubuntu Pro is free for personal use on up to 3 machines.")),
+            Text(_("Ubuntu Pro is free for personal use on up to three machines.")),
```

Comment:
@lechsandecki do you know if we have other occurrences of this text outside of the server installer?

Reply:
According to this it's five machines now https://ubuntu.com/pro/subscribe

Reply:
Updated to five.